src/bridge/lib/versions.py

```diff
@@ -3,7 +3,7 @@
 CONSUL_TEMPLATE_VERSION = "0.29.5"
 CONSUL_VERSION = "1.14.0"
 DAGSTER_VERSION = "1.0.17"
-KEYCLOAK_VERSION = "22.0"
```

Comment:
This will break the plugins that we are installing because Keycloak 20 is using an older version of Java than what the plugin Jars were compiled with.